src/components/TodoListFolder/TodoList.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import './TodoList.scss';
+import { Todo } from '../Todo';
+import { prepearedTodosType } from '../UserFolder';
+
+export function TodoList({ todos }) {
+  return (
+    <>
+      {todos.map(todo => (
+        <div
+          key={todo.id}
+          className={classNames(`row`,
+            { even: todo.id % 2 === 0 },
+            { odd: todo.id % 2 !== 0 })}
```

Comment:
Тебе нужен только один из этих классов. К примеру `even` классы будут с каким-то фоном, а у всех остальных обычных(odd) классов будут стили, которые ты сейчас задаешь через `odd`
```suggestion
          className={classNames(`row`,
            { even: todo.id % 2 === 0 },
          )}
```